docs/api.rst

```diff
@@ -733,6 +733,12 @@ section generates a new :class:`URL` instance.
       Support subclasses of :class:`int` (except :class:`bool`) and :class:`float`
       as a query parameter value.
 
+.. method:: URL.without_query_params(*keys)
```

Comment:
The signature or description should probably explain that the args are of type `str`: https://yarl--898.org.readthedocs.build/en/898/api/#yarl.URL.without_query_params.

Reply:
Also, the implementation does not call that argument `keys` so the docs shouldn't either.